src/components/LastTransaction/Knob.js

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components/macro'
+
+export default function Knob({ type, value }) {
+  const colors = getColors(type)
+  return (
+    <Wrapper data-testid={type} colors={colors}>
+      {value}
+    </Wrapper>
+  )
+
+  function getColors(type) {
+    const color = {
+      Grow: function () {
+        return [
+          'var(--main-yellow)',
+          'var(--main-darkerYellow)',
+          'var(--navigationButton-font-darkgrey)',
+        ]
+      },
+      Share: function () {
+        return [
+          'var(--main-green)',
+          'var(--main-darkerGreen)',
+          'var(--navigationButton-font-darkgrey)',
+        ]
+      },
+      Spend: function () {
+        return ['var(--main-blue)', 'var(--main-darkerBlue)', 'white']
+      },
+      default: function () {
+        return ''
+      },
+    }
+    return (color[type] || color['default'])()
+  }
+}
+
+const Wrapper = styled.div`
```

Comment:
One `Knob` is one section, so let's use a sectioning element instead of a div...

Reply:
Good point.